atlas_densities/app/cell_densities.py

```diff
@@ -817,6 +820,10 @@ def fit_average_densities(
     L.info("Loading average densities dataframe ...")
     average_densities_df = pd.read_csv(average_densities_path)
     homogenous_regions_df = pd.read_csv(homogenous_regions_path)
```

Comment:
same here, can't we update the input files to have the correct names?

Reply:
Input files updated.